.travis.yml

```diff
@@ -11,11 +11,22 @@ php:
   - 7.2
   - nightly
 
-allowed_failures:
-  - nightly
+before_install:
+  - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{,.disabled} || echo "xdebug not available"
+  - travis_retry composer self-update
 
-install:
-  - composer install -n
+install: travis_retry composer update --prefer-dist -n
 
 script:
   - vendor/bin/phpunit
+
+jobs:
+  include:
+    - stage: Code Quality
+      env: CODING_STANDARDS
+      php: 7.1
+      script:
+        - ./vendor/bin/phpcs lib tests
```

Comment:
We don't really need to specify the directories here since they're already defined in the config file

Reply:
Good point, I wonder from which repo I took this from. 😈